exporter.py

```diff
@@ -0,0 +1,228 @@
+from __future__ import annotations # Needed in order for something to reference itself in 'typing'
```

Comment:
should never put code above the license!

Reply:
IIRC this line has to be the first one in the file, see https://stackoverflow.com/questions/33837918/type-hints-solve-circular-dependency

But I haven't tried otherwise, so maybe I am mistaken !

_Sent from my Redmi Note 5 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

Reply:
Should work :) It's just an include statement, the commented license above is not affected by it